src/screens/DiscoveryScreenV2/index.js

```diff
@@ -311,13 +311,13 @@ const DiscoveryScreenV2 = ({route}) => {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
     flex: 1,
     paddingTop: 60
   },
   fragmentContainer: {
     height: '100%',
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintaining consistency in color usage across the application. By using `COLORS.white` instead of hardcoding the color value, it becomes easier to manage and update the color scheme of the application from a single place.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```
```diff
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```
This change also improves maintainability as any future changes to the color scheme can be done in the `theme` module, and those changes will be reflected throughout the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->